### PR TITLE
Add wasd

### PR DESCRIPTION
**Pull request by Meiallu**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/wasd/manifest.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  "nice_name": "Wasd",
+  "direct_ip": "wasdmc.com",
+  "server_wildcards": [],
+  "supported_languages": [
+    "pt-BR",
+    "pt-PT"
+  ],
+  "social": {
+    "web": "https://loja.wasdmc.com",
+    "web_shop": "https://loja.wasdmc.com"
```
Comment (maaaathis) [inline minecraft_servers/wasd/manifest.json:12]: use a specific link to your shop here

File: minecraft_servers/wasd/manifest.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
+{
+  "server_name": "wasd",
+  "nice_name": "Wasd",
+  "direct_ip": "wasdmc.com",
+  "server_wildcards": [],
```
Comment (DoJapHD) [inline minecraft_servers/wasd/manifest.json:5]: If you don't have any, remove

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/wasd/manifest.json (hunk 1 above)
Comment (DoJapHD) [reply minecraft_servers/wasd/manifest.json:10]: This is still the case
Change to https://loja.wasdmc.com/itens

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by DoJapHD** [comment]
You can re-add your Discord server now
Our script was broken but it should be fixed now

**Comment by Meiallu** [comment]
Interesting, the biggest image that I uploaded is 200kb.

**Comment by DoJapHD** [comment]
> Interesting, the biggest image that I uploaded is 200kb.

Yeah, the script checks all files not only the changed ones and with the new check I implemented this only came out now that there was something wrong

**Comment by Meiallu** [comment]
Is everything right, then?

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**